pkg/kubelet/dockertools/convert.go

```diff
@@ -74,3 +80,31 @@ func toRuntimeImage(image *docker.APIImages) (*kubecontainer.Image, error) {
 		Size: image.VirtualSize,
 	}, nil
 }
+
+// convert ContainerStatus to api.ContainerStatus.
+func containerStatusToAPIContainerStatus(containerStatus *kubecontainer.ContainerStatus) *api.ContainerStatus {
```

Comment:
For the DockerPrefix, we can just use `String()` of `containerStatus.ID`, if we want to move this function to runtime package in the future.

However, for now it seems that we couldn't move it to runtime package.
Because we directly set exited container to `Terminated` state here, but in fact some of them should be in `Waiting` state. They are just failed to be restarted in last `SyncPod()` and should be restarted in the future. For those containers, we'll set them to `Waiting` with the help of *reason cache* in the following part of `ConvertPodStatusToAPIPodStatus()` (Although there seems to be a little bug in that part now #17971).

I'm not sure whether rkt has similar logic like this. @yifan-gu 

And we also plan to move the reason cache logic up to Kubelet layer #18172 , after that I think we'll never need these kinds of convert functions in runtime interface any more. :)

/cc @yujuhong